pilot/main.py

```diff
@@ -39,16 +43,19 @@ def init():
     ask_feedback = True
     project = None
     run_exit_fn = True
+
+    args = init()
+
     try:
         # sys.argv.append('--ux-test=' + 'continue_development')
 
-        args = init()
-
         builtins.print, ipc_client_instance = get_custom_print(args)
 
-
         if '--api-key' in args:
             os.environ["OPENAI_API_KEY"] = args['--api-key']
+        if '--api-endpoint' in args:
```

Comment:
Also nto related but crept in - allow VSCode extension to specify the API endpoint.